### ISSUE TITLE
Clay 1.x, remove fieldset workaround for firefox

### ISSUE THREAD

**Issue by jgarcialr** [liferay/clay#3958]
Inside file _forms.scss
we have this piece of code to workaround fieldset on firefox
```
	@-moz-document url-prefix() { // FF Fieldset workaround
		display: table-cell;
	}
```
From firefox v86 this piece of code makes fieldsets and their childs not getting 100% width from it's parents.

If you remove this piece of code, it works as expected.
(LPS-128384)
Clay version   v.1.x

**Comment by pat270** [comment]
I cut a new version of 1.x should be under lexicon-ux@1.0.31.

@jbalsas @wincent There are some issues with the release script that I'm going to fix and send a pr for. One of the major issues I found was that the script creates a 1.x-develop branch which we phased out some time ago. Can one of you delete it from the repo? I don't think I have permissions to do that.

**Comment by wincent** [comment]
> One of the major issues I found was that the script creates a 1.x-develop branch which we phased out some time ago. Can one of you delete it from the repo? I don't think I have permissions to do that.

Done, but you do have permission to do it; if you have rights to push, that means you have the rights to modify branches (ie. not just adding commits, but force-updating branches and deleting them too).

How:

Fetch all the changes with `git remote update --prune`.

Inspect the branches to see what is pointing where. For this, I used `git graph upstream-rw/1.x{,-develop,-stable}*` (where `graph` is [an alias](https://github.com/wincent/wincent/blob/f71806fbb6217ad44517c661d1b35cee023342b6/aspects/dotfiles/templates/.gitconfig.erb#L30) for the rather horrid, `git --paginate log --graph --pretty=format:'%C(auto)%h%Creset%C(auto)%d%Creset %s %C(magenta bold)(%cr)%Creset %C(cyan)<%aN>%Creset'` — see the screenshot further down).

Given that the branches look to be in reasonable places (ie. `1.x-develop` pointing at the `f25b9b145` which is the `HEAD` of `1.x`, and `1.x-stable` very stale and pointing off at `30a616772` which is the last release from over 2 years ago), do a dry-run of the deletion:

```
git push upstream-rw :1.x-develop :1.x-stable --dry-run
To github.com:liferay/clay.git
 - [deleted]             1.x-develop
 - [deleted]             1.x-stable
```

Looks safe, so proceed to do the real thing:

```
git push upstream-rw :1.x-develop :1.x-stable                                                                                                                                                               
To github.com:liferay/clay.git
 - [deleted]             1.x-develop
 - [deleted]             1.x-stable
```

(Note getting rid of `1.x-stable` too to match what we've been doing in all our other repos for a while now.)

Finally, check the graph again to confirm that everything is pointing where we want:

```
git graph upstream-rw/1.x
* f25b9b145 (upstream/1.x, upstream-rw/1.x) Rebuild v1.0.31 (11 hours ago) <Patrick Yeo>
* a64553b15 Release v1.0.31 (11 hours ago) <Patrick Yeo>
* 125f45941 Update changelog (11 hours ago) <Patrick Yeo>
*   49a46c04a Merge pull request #3964 from pat270/clay-3958 (11 hours ago) <Patrick Yeo>
|\
| * 01319dddb fix(lexicon-css): Remove firefox `display: table-cell` CSS reset for `fieldsets` (12 hours ago) <Jorge García Jiménez>
|/
* 30a616772 Rebuild v1.0.30 (2 years, 1 month ago) <Patrick Yeo>
```

Basically the same thing but with color:

<img width="1099" alt="Screenshot 2021-03-02 -094137-IR8Ievkk@2x" src="https://user-images.githubusercontent.com/7074/109621658-8d5e8180-7b3b-11eb-9e11-78dee9ca8439.png">

One thing to note about all this is that even though the operations look pretty destructive, as long as you are careful and keep a record of what you are doing in your scrollback buffer, you can resurrect the deleted branches at will (because we have the hashes, all the same commits are still there, and branches are just pointers).

The other consequence of this is that anybody out there who does a fast-and-loose push from their local fork could conceivably recreate these old branches by accident. If that happens (it probably won't), we just re-delete.

**Comment by wincent** [comment]
@pat270: I updated the script (somewhat blindly) [here](https://github.com/liferay/clay/pull/3967).

**Comment by jgarcialr** [comment]
Hello I recompile portal with the new version and find that they are still fieldsets inside atlas, I'll open a follow-up issue and pr to get this fixed.
